<div class='pybot-out-usermsg'>PYBOT BETA VERSION 0 BUILD 118</div>
<div class='pybot-out-'>connected to database</div>
<div class='pybot-out-'>[PYBOT] Running with no filters</div>
<div class='pybot-out-'>[PYBOT] IRC object initialized, starting ping check...</div>
<div class='pybot-out-'>[PYBOT] Sending user info...</div>
<div class='pybot-out-'>[PYBOT] login success</div>
<div class='pybot-out-'>[PYBOT] Joining channel #exinmusic</div>
Unhandled exception in thread started by <bound method chatters.loopChatter of <irc.chatters instance at 0x7fc87adcfbd8>>
Traceback (most recent call last):
  File "/var/pybot/irc.py", line 44, in loopChatter
    chatInfo = self.getChatterInfo()
  File "/var/pybot/irc.py", line 37, in getChatterInfo
    data = json.loads(response.read())
  File "/usr/lib/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 384, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
